std/math.d

```diff
@@ -3979,6 +4061,14 @@ double fabs(double x) @safe pure nothrow @nogc { return fabs(cast(real) x); }
 ///ditto
 float fabs(float x) @safe pure nothrow @nogc { return fabs(cast(real) x); }
 
+///
+@safe unittest
+{
+    assert(fabs(0.0).approxEqual(0.0));
+    assert(fabs(-0.0).approxEqual(0.0));
+    assert(fabs(-10.0).approxEqual(10.0));
```

Comment:
In general I agree. Not for `fabs` though. There is no reasonable implementation that would do anything but change the sign bit, and the IEEE754-2008 standard prohibits doing anything else.
>The functionality closely follows the IEEE754-2008 standard for floating-point arithmetic, including the use of camelCase names rather than C99-style lower case names. All of these functions behave correctly when presented with an infinity or NaN.